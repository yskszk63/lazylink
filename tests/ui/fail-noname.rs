#[lazylink::lazylink]
mod libc {
    extern "C" {
        fn puts(v: *const i8);
    }
}

fn main() {}
