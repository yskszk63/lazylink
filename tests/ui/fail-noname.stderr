error: no name specified
 --> $DIR/fail-noname.rs:1:1
  |
1 | #[lazylink::lazylink]
  | ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
